maxim/test_runs/test_run_builder.py

```
@@ -377,10 +388,14 @@ def with_workflow_id(
             raise ValueError(
                 "yeilds_output is already set for this run builder. You can use either one of with_prompt_version_id, with_prompt_chain_version_id, with_workflow_id or yields_output in a test run."
             )
-        self._config.workflow = WorkflowConfig(id=workflow_id, context_to_evaluate=context_to_evaluate)
+        self._config.workflow = WorkflowConfig(
+            id=workflow_id, context_to_evaluate=context_to_evaluate
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)